README.md

```diff
@@ -73,3 +73,14 @@ vgrep supports the following commands:
 
 ## Files
 ![](screenshots/vgrep-files.png)
+
+## fzf
+
```

Comment:
Could you add some explanation here?  I have not heard of "fzf" until now and I guess there's an implicit assumption to copy the function below somewhere.   Having some guidance would be great.

Reply:
I've added some more description. If you would like to change anything I am open to it.